styleguide/js/zoolander.js

```diff
@@ -52,3 +52,19 @@ jQuery(document).ready(function() {
 
     $('#footer-accordion').tabCollapse();
 });
+
+/* Set the width of the side navigation to 250px and the left margin of the page content to 250px */
+function openNav() {
+    document.getElementById("navZoolander-container").style.width = "250px";
+    document.getElementById("mainContainer").style.marginLeft = "250px";
+}
+
+/* Set the width of the side navigation to 0 and the left margin of the page content to 0 */
+function closeNav() {
+    document.getElementById("navZoolander-container").style.width = "0";
+    document.getElementById("mainContainer").style.marginLeft = "0";
+}
+
+jQuery (".navZoolander-hasDropdown").hover(function(){
```

Comment:
Well in theory yes, but I'm wondering if JS cleanup should just be a separate task, because there are some other issues with how the code is written that may be worth looking at separately.  I'd say just change it here, and we can worry about the rest of the file at a later time.